xml/sec_images_hardened.xml

```diff
@@ -0,0 +1,240 @@
+     information, refer to <link
+      xlink:href="https://www.hhs.gov/hipaa/for-professionals/security/index.html"/>.
+    </para>
+   </listitem>
+  </varlistentry>
+  <varlistentry>
+   <term>Payment Card Industry Data Security Standard (<link xlink:href="https://github.com/ComplianceAsCode/content/blob/master/products/sle15/profiles/pci-dss.profile"><filename>pci-dss.profile</filename></link>)</term>
+   <listitem>
+    <para>
+     The <systemitem>Payment Card Industry Data Security Standard (PCI
+     DSS)</systemitem> is a set of requirements to guide merchants to protect
+     cardholder data. It is  maintained by the <orgname>PCI Security Standards
+     Council (SSC)</orgname> that was founded by all five major credit card
+     brands Visa, MasterCard, American Express, Discover, and JCB. For more
+     information, refer to <link
+      xlink:href="https://www.pcisecuritystandards.org/document_library"/>.
+    </para>
+   </listitem>
+  </varlistentry>
+ </variablelist>
+ <para>
+  All profile files are available in the <link
+   xlink:href="https://github.com/ComplianceAsCode/content/tree/master/products/sle15/profiles">ComplianceAsCode</link>
+  repository.
+ </para>
+ <para>
+  For a complete list of rules that have been applied during pre-hardening, refer
+  to <link
+   xlink:href="https://github.com/ComplianceAsCode/content/blob/master/products/sle15/profiles/pcs-hardening.profile"><filename>pcs-hardening.profile</filename></link>.
+  This profile is a combination of the <literal>STIG</literal> and
+  <literal>CIS</literal> profiles minus rules that can only be applied after
+  instance creation.
+ </para>
+ <important>
+  <title>Recommended profiles</title>
+  <para>
+   &suse; recommends using the either <literal>CIS</literal> or
+   <literal>STIG</literal> profile. You can use other profiles at your own
+   discretion.
+  </para>
+ </important>
+ <!--
+ <para>
+  Other policies are available on the &openscap; website at <link
+   xlink:href="https://www.open-scap.org/security-policies/choosing-policy/"/>.
+ </para>
+ -->
+ <para>
+  To evaluate an instance, you can run:
+ </para>
+<screen>&prompt.sudo;<command>oscap</command> xccdf eval \
+ --profile <replaceable>stig</replaceable><co xml:id="co-eval-profile"/> \
+ --results <replaceable>/tmp/results.xml</replaceable><co xml:id="co-eval-results"/> \
+ --report <replaceable>/tmp/report.html</replaceable><co xml:id="co-eval-report"/> \
+ --stig-viewer <replaceable>/tmp/stigviewer.xml</replaceable><co xml:id="co-eval-viewer"/> \
+ /usr/share/xml/scap/ssg/content/ssg-sle15-ds-1.2.xml<co xml:id="co-eval-ssg"/></screen>
+ <calloutlist>
+  <callout arearefs="co-eval-profile">
+   <para>
+    Specifies the profile to use, e.g. <literal>stig</literal> or
+    <literal>cis</literal>.
+   </para>
+  </callout>
+  <callout arearefs="co-eval-report">
+   <para>
+    Saves the results of the evaluation to <filename>/tmp/results.xml</filename>
```

Comment:
when I build this, it gives this an icon of "3" which is a mismatch for the "2" it matches above... I think the xml:id's are mixed up on lines 172, 173

Reply:
Fixed.